docs/source/tutorials/HLS/HLS_search.ipynb

```diff
@@ -83,7 +83,7 @@
    "id": "25576924",
    "metadata": {},
    "source": [
-    "## Creating an AOI\n",
+    "Creating an AOI: \n",
```

Comment:
@sumantjha why did you remove the markdown heading?

Reply:
That was causing the TOC headings in the html. I double checked by running Sphinx on local. Without the markdown heading, the TOC looks as it should.

Reply:
> We want those headings in the TOC, example: ![image](https://user-images.githubusercontent.com/306179/195147064-11c8a850-1286-496b-854b-6b9477d03843.png)

Done!